modules/restore-regenerate-etcd-certs.adoc

```diff
@@ -0,0 +1,173 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-regenerate-etcd-certs_{context}"]
+= Regenerating etcd certificates
+
+If the etcd certificates have expired for your master host, you must regenerate them before you can add the member back to the etcd cluster.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+// * SSH access to the master host to add to the etcd cluster.
+// * The IP address of an existing active etcd member.
+
+// TODO: Figure out prereqs and assumptions. you've already removed it?
+
+.Procedure
+
+. TODO: Check whether the certs are expired or not. If they're valid, skip procedure
++
+// TODO: right now, the following steps are a straight copy/paste from restoring lost master hosts procedure (step 5)
+
+. Set up a temporary etcd certificate signer service on your master where you have restored etcd.
```

Comment:
. Set up a temporary etcd certificate signer service on one of the healthy master nodes.